### ISSUE TITLE
Runkeeper <> Strava sync down

### ISSUE THREAD

**Issue by johan4711** [cpfair/tapiriik#170]
It seems the sync between Runkeeper and Strava is down in both directions for me, my last activity was the 8th of August and I've since done three more in Runkeeper and one in Strava, none of which gets synced. Checking the https://tapiriik.com/activities page, none of these activities are listed in neither the Runkeeper nor Strava columns.

Does anyone experience this as well? Any ideas where to start looking?

**Comment by ilyaguy** [comment]
I have the same issue. Also, when I try to change settings, I get "502" error.

**Comment by W73** [comment]
Syncing Garmin Connect -> (strava, endomondo, runkeeper) 
I have queuing rotating and can not sync it automatically or manually. It happened when i removed smashrun, but it may be connected with something else. 
Last synced run 9th Aug 7pm CET. So it could be synced  between 8pm-9pm.

**Comment by gep13** [comment]
Yip, I have the same thing.

I have just removed all my services, and re-added them, but back to the same queuing message.

**Comment by alexmonteiro** [comment]
Same issue.

**Comment by W73** [comment]
It started working again. Thank you !!!

**Comment by johan4711** [comment]
Yes, mine was automatically synced too, great! Thanks! I guess this issue can be closed now.

**Comment by cuh7b5** [comment]
I had the same issue a few days ago. I removed / reconnected all my accounts and it helped. From yesterday I have the same issue again without touching anything. Tapiriik is queuing but not syncing any activities from Garmin to Runkeeper and Endomondo. Nothing appears in the activities list either.

**Comment by desrod** [comment]
Not working as of September 20th for any services in any direction which include Strava. Strava's API and some of the fields have changed for rides, which may be why things are breaking, but without access to the error logs, it's all just speculation.

**Comment by vpakg** [comment]
Possibly duplicated, see issue #155
